rero/reroils-data
# feat: add is_part_of field in brief and full document views

How to test:
- Search article
- Check if the "is part of" field is present on brief and full view

Signed-off-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>
